### DISCUSSION TITLE
2022-01-12 Weekly Meeting 38 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#161] [Weekly Meetings]
Weekly Meeting 38 is happening on Wednesday January 12th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates

- WASM Meeting, how did it go? Would any participant like to represent the group and give a summary?
- Next website meeting is on Tuesday, January 18th
- OMI-Chan status?
- We have a discussion (and milestone) around the future of OMI. We should discuss what that could look like. Started listing known tasks and responsibilities which could help inform future chair structure, working groups, operations, etc. We want to properly divide workload to reduce bottlenecks and improve sustainability. **++ On that note, next election is 2/2/22**
- From @LightLodges: discussions/milestones = EVENTS and community driven engagement strategies including cadence and what we're ready to aim for in budgets/planning. Let's talk events!
- OMI glTF extensions group update

Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
+ check in with gltf group 
+ OMI-Chan status? 
+ Interoperators effort set for possibly this Friday (Evo) 
+ Backlog refinement public meeting every 3rd Tuesday of month at 22:05UTC 
+ Updated readme to reflect working groups, ensure your group is up to date with link to repo and time you meet (if appropriate) 
+ We have a discussion (and milestone) around the future of OMI. We should discuss what that could look like. Started listing known tasks and responsibilities which could help inform future chair structure, working groups, operations, etc. We want to properly divide workload to reduce bottlenecks and improve sustainability. 
++ On that note, next election is 2/2/22

**Reply by mrmetaverse** [reply]
Next website meeting is next week on Tuesday the 18th, what about others?

**Reply by mrmetaverse** [reply]
Also, thanks for setting this up @antpb!

**Comment by LightLodges** [comment]
Agenda item related to discussions/milestones = EVENTS and community driven engagement strategies including cadence and what we're ready to aim for in budgets/planning.

Streams - regularly? Weekly?
Demos - quarterly? Next one in March?
Hackathons - April? June or July partnered events with others?

There's interest in partnering up to make sure the hackathons start to happen by spring. Are we do to partner up with others engaged in hackathon building for interop and WebXR for spring and summer, and do we have a clear sense of how those relationships need to work for everyone to benefit?

**Comment by mrmetaverse** [comment]
WASM group update: 

- https://github.com/omigroup/omi-wasm-group/discussions/9
- Next meeting is next Tuesday 
- Hope you will get involved

**Comment by mrmetaverse** [comment]
Website: 

- Robert got a first draft finished last week

**Comment by mrmetaverse** [comment]
Future structure of OMI: Very important conversation to be had as a group before next election. Do we need more chairs, new ways of dividing work? 

Notes on known responsibilities of OMI leadership: https://hackmd.io/@mrmetaverse/omi-tasks-audit 

How might we use this list (and add to it) to codify potential roles for the future structure?

**Comment by mrmetaverse** [comment]
RE: From @LightLodges: discussions/milestones = EVENTS and community driven engagement strategies including cadence and what we're ready to aim for in budgets/planning. Let's talk events!

- expand our open community events 
- help people find out what the open metaverse is working on and how to get involved 
- showing up in others communities 
- inviting people in to our community
- possibly physical meetings as well
- hackathons, etc. 
- Metatraversal III
  - Next meetup for that is happening tomorrow (Thursday 18:30 UTC ) for questions ping Evo on Discord :) 
  - identity management, avatar solutions, questions around metatraversal that require conversation and not assumptions 
- Where do you think we should be putting our attention in terms of education and development 
  
past events: [OMI Interoperability Demo Night](https://youtu.be/e26UJRCGfGk)

**Comment by mrmetaverse** [comment]
New updates for Audio emitter explainer, new PR available. 
- list contributors 
- next meeting is Thursday at 22:30UTC - 23:30UTC
- ex of a great way to share utility for a PR that OMI puts out: https://github.com/mozilla/hubs/pull/4741#issuecomment-1004431004 (see comments)
